lib/api/helpers.js

```diff
@@ -1,10 +1,36 @@
 import url from "url";
-import { slugify } from "transliteration";
 import { Meteor } from "meteor/meteor";
 import { check } from "meteor/check";
 import { Router } from "/imports/plugins/core/router/lib";
 import { Shops } from "/lib/collections";
 
+// Array of ISO Language codes for all languages that use latin based char sets
+// list is based on this matrix http://w3c.github.io/typography/gap-analysis/language-matrix.html
+// list of lang codes https://en.wikipedia.org/wiki/List_of_ISO_639-1_codes
+export const latinLangs = ["az", "da", "de", "en", "es", "ff", "fr", "ha", "hr", "hu", "ig", "is", "it", "jv", "ku", "ms", "nl", "no", "om", "pl", "pt", "ro", "sv", "sw", "tl", "tr", "uz", "vi", "yo"]; // eslint-disable-line max-len
```

Comment:
Is this something we could / should store in the DB somewhere? Or maybe it's something that's already stored? We could add a `isLatin` flag on stored languages?

I'd think we wouldn't want anyone to have to come and edit this hardcoded list if they add a new language.